### PR TITLE
[refs #387] [generic.reset] Add better default

### PR DESCRIPTION
**Pull request by florianbouvot**

### CODE REVIEW

File: generic/_generic.reset.scss [hunk 1]
```diff
@@ -33,24 +76,19 @@ li > {
...
 
 
-
 /**
- * Remove default table spacing.
+ * Prevent double borders.
  */
 
-table {
+ table {
```
Comment (yourownmood) [inline generic/_generic.reset.scss]: There is a extra space before the selector 👾

Comment (florianbouvot) [reply generic/_generic.reset.scss]: Thank you

**Comment by florianbouvot** [comment]
Why not moving `table` declaration to `_elements.tables.scss`?

**Comment by herzinger** [comment]
If it's a table reset, it should stay in `generics.reset`. Leave the elements layer for more opinionated styling.

File: generic/_generic.reset.scss [hunk 2]
```diff
@@ -3,20 +3,63 @@
...
+ * Add the correct display in IE.
  */
 
-body,
-h1, h2, h3, h4, h5, h6,
-blockquote, p, pre,
-dl, dd, ol, ul,
+main {
+  display: block;
+}
```
Comment (csshugs) [inline generic/_generic.reset.scss:11]: If we don’t want to support any IE anymore with version 7, I guess we don’t need this reset.

**Review by csshugs — CHANGES_REQUESTED** [review]

**Review by csshugs — CHANGES_REQUESTED** [review]
A changelog entry is missing.

File: generic/_generic.reset.scss (hunk 2 above)
Comment (florianbouvot) [reply generic/_generic.reset.scss:11]: Agree

Comment (csshugs) [reply generic/_generic.reset.scss:11]: Scratch that. No version of IE (not even IE11) does support the `<main>` element. So we better leave this in.

**Review by csshugs — APPROVED** [review]

**Pull request merged**